### PR TITLE
Makes language staff focused

### PR DESCRIPTION
**Pull request by clausmullie**
Works towards #1347

### CODE REVIEW

**Comment by clausmullie** [comment]
Working my way down, got as far as staff-information

**Comment by clausmullie** [comment]
Likely still some work to be done, but this moves a lot of sentences onto separate lines and tweaks some content here and there.

File: README.md [hunk 1]
```diff
@@ -3,11 +3,14 @@
...
 <!-- SPDX-FileCopyrightText: 2018-2023 The Foundation for Public Code <info@publiccode.net> -->
 
+We help open source projects for public organizations to become successful, build sustainable communities around them and create a thriving public open source ecosystem.
+
 [Our mission](organization/mission.md) is:
 
 > [...] to enable public-purpose software and policy that is open and collaborative.
 
-This repository is meant to reflect the concept of the Foundation for Public Code and hold all information about it.
+The [About site](activities/documentation/index.md) is home to all of our staff information, decision-making rules and processes. It is our staff manual that can be developed collaboratively with the community and reused by everyone.
```
Comment (Ainali) [inline README.md:12]: Perhaps one sentence per line here?
```suggestion
The [About site](activities/documentation/index.md) is home to all of our staff information, decision-making rules and processes.
It is our staff manual that can be developed collaboratively with the community and reused by everyone.
```

File: activities/communication/basic-tweet-template.md [hunk 2]
```diff
@@ -14,8 +14,10 @@ Our tweets are often accompanied by relevant images or emoji.
 
 ## When to use this template
 
-When suggesting something for social sharing, use this template to create a draft tweet or toot (for Mastodon). Also consider images, gifs or emoji. Note that Twitter's algorithm prioritizes tweets with images or video.
+When suggesting something for social sharing, use this template to create a draft tweet or toot (for Mastodon).
+Also consider images, gifs or emoji. Note that Twitter's algorithm prioritizes tweets with images or video.
```
Comment (Ainali) [inline activities/communication/basic-tweet-template.md:18]:
```suggestion
Also consider images, gifs or emoji.
Note that Twitter's algorithm prioritizes tweets with images or video.
```

**Comment by clausmullie** [comment]
Yes, that would have been the right way to do it. I started out with an intention of focusing on language, started picking up the one lines along the way, and the latter then spiraled into becoming the bulk of the changes :see_no_evil:

File: activities/communication/how-to-post-to-social-media.md [hunk 3]
```diff
@@ -32,7 +34,7 @@ Most of our social followers are real people (many of whom we know!) who are alr
 
 ## Tips for writing social media posts
 
-* The most interesting posts are the ones from our community and codebases. Try to showcase their success stories before Foundation for Public Code centered news.
+* The most interesting posts are the ones from our community and codebases.Try to showcase their success stories before Foundation for Public Code centered news.
```
Comment (Ainali) [inline activities/communication/how-to-post-to-social-media.md:37]:
```suggestion
* The most interesting posts are the ones from our community and codebases. Try to showcase their success stories before Foundation for Public Code centered news.
```
This space is needed.

File: activities/creating-affiliations/index.md [hunk 4]
```diff
@@ -11,8 +11,6 @@ Other organizations that we collaborate with can become [affiliates](../../gloss
 
 [See our current affiliates](../../organization/affiliates.md).
 
-If you're interested in being affiliated with us, or have any questions, please email us at <info@publiccode.net>.
```
Comment (Ainali) [inline activities/creating-affiliations/index.md:14]: Why is this removed? We still want organizations as affiliates, right?

File: activities/financial-administration/reporting.md [hunk 5]
```diff
@@ -36,12 +39,16 @@ The financial statement must be published on our website within six months after
...
 
-The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget. The report must include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our drive](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj).
+The report should be signed by an authorized officer, and the financial statement should be attested by the responsible financial officer or a certified public accountant, and should reflect the categories stipulated in the attached approved budget.
+The report must include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received. [Detailed financial requirements can be found on our drive](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj).
```
Comment (Ainali) [inline activities/financial-administration/reporting.md:54]:
```suggestion
The report must include a reconciliation of all amounts spent for the purpose specified, with the total grant proceeds received.
[Detailed financial requirements can be found on our drive](https://drive.google.com/drive/folders/1Tv5j1vfn2MhDc2DtDmKCyQrrfkh2CJTj).
```

File: activities/index.md [hunk 6]
```diff
@@ -6,10 +6,6 @@ type: Index
 
 # Activities
 
-Higher level activities of the Foundation for Public Code that directly serve to further the [mission](../organization/mission.md):
```
Comment (Ainali) [inline activities/index.md:9]: This explainer is still true, right? And it feels helpful as it explains why we haven't an activity for every little thing we do.

File: activities/staff-management/contracts.md [hunk 7]
```diff
@@ -7,15 +7,23 @@ explains: how contracts and contract renewals are managed
...
 
 ## Employment contract renewals
 
-The operations coordinator keeps track of the contract expiration dates [in a spreadsheet](https://docs.google.com/spreadsheets/d/1isVOEetjiaLKMxJHPGPusnBLORxjHUc_/edit#gid=21927936). The operations coordinator sends a reminder to the executive team about an upcoming renewal 6 weeks in advance, to allow 2 weeks for discussion before the one month notice period.
+The operations coordinator keeps track of the contract expiration dates [in a spreadsheet](https://docs.google.com/spreadsheets/d/1isVOEetjiaLKMxJHPGPusnBLORxjHUc_/edit#gid=21927936).
+The operations coordinator sends a reminder to the executive team about an upcoming renewal 6 weeks in advance, to allow 2 weeks for discussion before the one month notice period.
 
-Employees are told whether their contract will be renewed one month before it expires. This gives them one month's notice if the contract is not renewed. One month's notice is the customary in The Netherlands.
+Employees are told whether their contract will be renewed one month before it expires.
+This gives them one month's notice if the contract is not renewed. One month's notice is the customary in The Netherlands.
```
Comment (Ainali) [inline activities/staff-management/contracts.md:22]:
```suggestion
This gives them one month's notice if the contract is not renewed.
One month's notice is the customary in The Netherlands.
```

File: activities/staff-management/contracts.md [hunk 8]
```diff
@@ -26,5 +34,6 @@ The process is as follows:
 
 ## Service agreements and contractors
 
-We work with contractors and freelancers. For big or long-term assignments a service agreement is drafted. For small or short-term assignments a quote suffices.
+We work with contractors and freelancers. For big or long-term assignments a service agreement is drafted.
```
Comment (Ainali) [inline activities/staff-management/contracts.md:37]:
```suggestion
We work with contractors and freelancers.
For big or long-term assignments a service agreement is drafted.
```

File: activities/staff-management/sick-leave.md [hunk 9]
```diff
@@ -12,28 +12,42 @@ Staff report that they are sick as per our [sickness policy](../staff-informatio
...
+A case manager from RegioPoortwachters will reach out to inquire about the sick employee approximately 10 days after the initial sickness report in the insurance portal.
+The ARBO service portal is integrated with our insurance portal.
 
-The ARBO service will plan a meeting between the employee and an ARBO doctor to assess the employee's situation. Based on the assessment, the employee will start reintegration when their situation allows. The ARBO service will draft a [reintegration plan (plan van aanpak)](https://business.gov.nl/regulation/working-conditions-employees/) for the employee. This plan should include the procedures all involved parties must follow. The chief executive must approve the plan.
+The ARBO service will plan a meeting between the employee and an ARBO doctor to assess the employee's situation.
+Based on the assessment, the employee will start reintegration when their situation allows.
+The ARBO service will draft a [reintegration plan (plan van aanpak)](https://business.gov.nl/regulation/working-conditions-employees/) for the employee.
+This plan should include the procedures all involved parties must follow. The chief executive must approve the plan.
```
Comment (Ainali) [inline activities/staff-management/sick-leave.md:34]:
```suggestion
This plan should include the procedures all involved parties must follow.
The chief executive must approve the plan.
```

File: activities/staff-management/sick-leave.md [hunk 10]
```diff
@@ -12,28 +12,42 @@ Staff report that they are sick as per our [sickness policy](../staff-informatio
...
 
-If the employee has been sick for more than three quarters of a year, [it must be reported to the UWV](https://www.uwv.nl/werkgevers/werknemer-is-ziek/loondoorbetaling/werknemer-is-langdurig-ziek/index.aspx). The 42nd week report must be submitted no later than the first working day after the 42nd week of illness.
+If the employee has been sick for more than three quarters of a year, [it must be reported to the UWV](https://www.uwv.nl/werkgevers/werknemer-is-ziek/loondoorbetaling/werknemer-is-langdurig-ziek/index.aspx).
+The 42nd week report must be submitted no later than the first working day after the 42nd week of illness.
 
-If the employee has not completed reintegration after a year, a one year evaluation must take place. During this evaluation the director and the employee discuss whether the integration approach has been successful, and a report is prepared. The ARBO case manager will be available for guidance and instructions.
+If the employee has not completed reintegration after a year, a one year evaluation must take place. During this evaluation the director and the employee discuss whether the integration approach has been successful, and a report is prepared.
```
Comment (Ainali) [inline activities/staff-management/sick-leave.md:52]:
```suggestion
If the employee has not completed reintegration after a year, a one year evaluation must take place.
During this evaluation the director and the employee discuss whether the integration approach has been successful, and a report is prepared.
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 11]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
 
 ## Preparation
 
-Put big sheets of paper on the tables the exercise will be held on. This allows for the groups to draw explanations on them. The building blocks can also taped to them so that everything can be saved. Cut a lot of arrows, both one way and double sided in an array of colors. Use different colors for the current and ideal parts of the exercise so less meaning is inherited between them. Print and cut icons that are used for the actors. Print and cut the problem cards.
+Put big sheets of paper on the tables the exercise will be held on.
+This allows for the groups to draw explanations on them.
+The building blocks can also taped to them so that everything can be saved.
+Cut a lot of arrows, both one way and double sided in an array of colors.
+Use different colors for the current and ideal parts of the exercise so less meaning is inherited between them.
+Print and cut icons that are used for the actors. Print and cut the problem cards.
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:32]:
```suggestion
Print and cut icons that are used for the actors.
Print and cut the problem cards.
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 12]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
...
 
-Imagine that you all are living and working in Middletown **municipality**. It is of decent size, not too small, but also not one of the biggest.
+Imagine that you all are living and working in Middletown **municipality**.
+It is of decent size, not too small, but also not one of the biggest.
 
-Together with **three other municipalities** you have developed an **open source IT system**. It is a festival management system. You use it for managing festivals taking place in your respective municipalities. It has been in use for two years. The main **end users** of the system are civil servants in your municipality.
+Together with **three other municipalities** you have developed an **open source IT system**.
+It is a festival management system.
+You use it for managing festivals taking place in your respective municipalities.
+It has been in use for two years. The main **end users** of the system are civil servants in your municipality.
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:46]:
```suggestion
It has been in use for two years.
The main **end users** of the system are civil servants in your municipality.
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 13]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
...
-Each of you have a separate deployment of the system. Two each with different **hosting vendors** and two have deployed on their own servers.
+Each of you have a separate deployment of the system.
+Two each with different **hosting vendors** and two have deployed on their own servers.
 
 **Citizens** can interact with some of the content from the system through websites about the festivals.
 
 ### Exercise 1
 
-Try to map out a likely way that this is being _maintained_. Include both strengths and weaknesses. That is, try to place the actors involved actors and show what is happening between them.
+Try to map out a likely way that this is being _maintained_. Include both strengths and weaknesses.
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:58]:
```suggestion
Try to map out a likely way that this is being _maintained_.
Include both strengths and weaknesses.
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 14]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
...
+That is, try to place the actors involved actors and show what is happening between them.
 
-Feel free to draw upon your own experience and prejudice about municipal work. If you don't know, make an educated guess.
+Feel free to draw upon your own experience and prejudice about municipal work.
+If you don't know, make an educated guess.
 
-Use the available icons and arrows to show what actors are involved. What are the relationships? Show:
+Use the available icons and arrows to show what actors are involved.
+What are the relationships? Show:
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:65]:
```suggestion
What are the relationships?
Show:
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 15]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
...
+Write on the arrows to make clear what they mean.
+You are encouraged to write on the icons to help you distinguish them better.
+If you need more icons or arrows, ask.
 
 ### Exercise leader notes
 
-To get them started, ask them to start placing a few actors on the paper and discuss their relationships. Make sure the groups move forward and take decisions and don't get stuck in details. Remind them of the time constraint. When they have some actors and a few arrows down, start introducing the problem cards. If a group needs to be pushed a little bit, ask them who is involved, who things get reported to and so on. Make sure they write somewhere what the arrows mean.
+To get them started, ask them to start placing a few actors on the paper and discuss their relationships.
+Make sure the groups move forward and take decisions and don't get stuck in details.
+Remind them of the time constraint. When they have some actors and a few arrows down, start introducing the problem cards.
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:79]:
```suggestion
Remind them of the time constraint.
When they have some actors and a few arrows down, start introducing the problem cards.
```

File: activities/supporting-codebase-governance/exercise/run-governance-exercise.md [hunk 16]
```diff
@@ -22,69 +24,102 @@ Each of these should have an introduction to the task and explain the expectatio
...
+If a group needs to be pushed a little bit, ask them who is involved, who things get reported to and so on. Make sure they write somewhere what the arrows mean.
```
Comment (Ainali) [inline activities/supporting-codebase-governance/exercise/run-governance-exercise.md:80]:
```suggestion
If a group needs to be pushed a little bit, ask them who is involved, who things get reported to and so on.
Make sure they write somewhere what the arrows mean.
```

File: activities/tool-management/github.md [hunk 17]
```diff
@@ -6,15 +6,18 @@ type: resource
...
 
 ## Using GitHub
 
 GitHub is our bread and butter when it comes to codebase management, codebase auditing, issue tracking and version control, both for ourselves and our community.
 
 ## GitHub management
 
-Our GitHub account is managed by the directors and codebase stewards. New employees are onboarded by one of the codebase stewards. We also have [documention available on About for newcomers](../trainings/github-for-newcomers.md).
+Our GitHub account is managed by the directors and codebase stewards.
+New employees are onboarded by one of the codebase stewards. We also have [documention available on About for newcomers](../trainings/github-for-newcomers.md).
```
Comment (Ainali) [inline activities/tool-management/github.md:20]:
```suggestion
New employees are onboarded by one of the codebase stewards.
We also have [documention available on About for newcomers](../trainings/github-for-newcomers.md).
```

File: activities/tool-management/index.md [hunk 18]
```diff
@@ -6,7 +6,13 @@ type: index
 
 # Tool management and guidance
 
-The Foundation for Public Code needs tools to accomplish its mission. We use a variety of digital tools. Our tools range from being entirely self-managed, having service providers, to being completely externally managed. The operations coordinator is the system administrator of all the self-managed tools. They can create accounts and ensure updates happen regularly. The chief executive can do this as well. Most of these tools are used for our internal processes. Some are available to our community.
+The Foundation for Public Code needs tools to accomplish its mission.
+We use a variety of digital tools. Our tools range from being entirely self-managed, having service providers, to being completely externally managed.
```
Comment (Ainali) [inline activities/tool-management/index.md:10]:
```suggestion
We use a variety of digital tools. Our tools range from being entirely self-managed, having service providers, to being completely externally managed.
```
```suggestion
We use a variety of digital tools.
Our tools range from being entirely self-managed, having service providers, to being completely externally managed.
```

File: activities/tool-management/mattermost.md [hunk 19]
```diff
@@ -6,15 +6,17 @@ type: resource
...
+Mattermost chat is an important means of communication for the team.
 
 ## Using Mattermost
 
 [Read our chat guidance](https://about.publiccode.net/activities/communication/using-chat.html).
 
 ## Mattermost management
 
-Mattermost is installed, maintained and kept up-to-date by the operations coordinator and is running on a VPS with 8 GB of RAM. User registration and management also falls under the operations coordinator's responsibilities. This includes uploading important emojis :)
+Mattermost is installed, maintained and kept up-to-date by the operations coordinator and is running on a VPS with 8 GB of RAM. User registration and management also falls under the operations coordinator's responsibilities.
```
Comment (Ainali) [inline activities/tool-management/mattermost.md:18]:
```suggestion
Mattermost is installed, maintained and kept up-to-date by the operations coordinator and is running on a VPS with 8 GB of RAM.
User registration and management also falls under the operations coordinator's responsibilities.
```

File: organization/partnerships.md [hunk 20]
```diff
@@ -6,6 +6,7 @@ type: resource
 
 # Partnerships
 
-The Foundation for Public Code is a member-owned association that exists in an ecosystem of partners and [affiliates](affiliates.md). Our partners are integral to our ability to [achieve our mission](https://about.publiccode.net/organization/mission.html). Read more about [becoming a partner](../activities/creating-partnerships/index.md).
+The Foundation for Public Code is a member-owned association that exists in an ecosystem of partners and [affiliates](affiliates.md). Our partners are integral to our ability to [achieve our mission](https://about.publiccode.net/organization/mission.html).
```
Comment (Ainali) [inline organization/partnerships.md:9]:
```suggestion
The Foundation for Public Code is a member-owned association that exists in an ecosystem of partners and [affiliates](affiliates.md).
Our partners are integral to our ability to [achieve our mission](https://about.publiccode.net/organization/mission.html).
```

File: activities/creating-partnerships/index.md [hunk 21]
```diff
@@ -18,7 +18,6 @@ The Foundation for Public Code is actively looking for diverse public organizati
 
 Only public organizations can become partners.
 Other organizations can become [affiliates](../creating-affiliations/index.md)
-If you're interested in partnership for your organization, or have any questions, please email us at <partnerships@publiccode.net>.
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/creating-partnerships/index.md:21]: Because this a page in the staff manual, so this sentence should be moved to a .net page

Or do you think it would be useful to still have these sign posts here and there?

[the remark this answers is not in the document]
Comment (Ainali) [reply activities/creating-partnerships/index.md:21]: It could be rephrased if it is internal facing, but we should still have documentation somewhere on About which email is used for what. 

Also, I don't recall the new design of the website having this info, so perhaps wait with removing it here until there is a plan to actually put it there?

[the remark this answers is not in the document]
Comment (ericherman) [reply activities/creating-partnerships/index.md:21]: Agreed. Until there is a partners page to link to on dot-net, let's phrase it something like, "The email list <partnerships@publiccode.net> is maintained to answer questions and provide information to Organizations interested in partnership." or similar.

[the remark this answers is not in the document]
Comment (ericherman) [reply activities/creating-partnerships/index.md:21]: > Or do you think it would be useful to still have these sign posts here and there?

I do, as that will likely still be useful for us, yet should have a change in tone in the wording.

File: activities/creating-affiliations/index.md (hunk 4 above)
Comment (clausmullie) [reply activities/creating-affiliations/index.md:14]: Sentence restored and reworded

File: activities/creating-partnerships/index.md (hunk 21 above)
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/creating-partnerships/index.md:21]: Good points. Sentence restored and reworded.

File: activities/index.md (hunk 6 above)
Comment (clausmullie) [reply activities/index.md]: Yes, restored to original. I'm wondering how process code will fit in here, and whether standard maintenance is not also a higher level activity.

My instinct was to flatten it for staff - as each staff member has their own priority, but fine to leave as is.

File: activities/index.md [hunk 22]
```diff
@@ -6,10 +6,6 @@ type: Index
...
-
-* [Codebase stewardship](codebase-stewardship/index.md): Enabling a codebase to attain the sustainability, marketability, support and quality needed to be successfully reused and community developed.
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/index.md]: Yes, restored to original. I'm wondering how process code will fit in here, and whether standard maintenance is not also a higher level activity.

My instinct was to flatten it for staff - as each staff member has their own priority, but fine to leave as is.

File: README.md [hunk 23]
```diff
@@ -18,11 +22,13 @@ It's `main` branch is the official ['source of truth'](GOVERNANCE.md) and centra
 
 We are looking for people like you to [contribute](CONTRIBUTING.md) to this project by suggesting improvements and helping develop the Foundation for Public Code. 😊
 
-Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms. Please be lovely to all other community members.
+Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md).
+By participating in this project you agree to abide by its terms. Please be lovely to all other community members.
```
Comment (ericherman) [inline README.md:26]: Linebreak between sentences?

Comment (clausmullie) [reply README.md]: Done! :)

**Review by Ainali — APPROVED** [review]
Perhaps a bit redundant to list codebase stewardship twice in the Activities index, but that is not a blocker.

**Review by ericherman — APPROVED** [review]
Ideally, this would be two separate PRs.
First, a big PR which does all the sentence-per-line changes, with no content changes.
Second, as small PR which is the actual sentence changes.

To verify that I had not over-looked a meaningful change, I used `git diff --ignore-all-space --word-diff  develop` from the command-line.

**Comment by clausmullie** [comment]
Yes, lesson learnt for next time

File: activities/index.md [hunk 24]
```diff
@@ -19,6 +19,7 @@ Activities that support the above as well as make staff operations work:
 * [Conflicts of interest](conflict-of-interest/index.md)
 * [Creating affiliations](creating-affiliations/index.md)
 * [Creating partnerships](creating-partnerships/index.md)
+* [Codebase stewardship](codebase-stewardship/index.md)
```
Comment (ericherman) [inline activities/index.md:22]: Indeed this is also above on line 11, pulled out special.

It's not _wrong_ to say that Codebase stewardship supports Codebase stewardship, although it feels a bit silly.

I suppose maybe the more important question is whether or not we'd miss it in the bullet list if we went looking for it.
